### DISCUSSION TITLE
Sending products and catalogs

### DISCUSSION THREAD

**Discussion by tarekkoura** [devlikeapro/waha#248] [Ideas]
Ca Waha Plus send products or catalogs from a Bussiness Account?

**Comment by allburov** [comment]
Hi! Not, right now it's not possible :( 
For now you can send a link to `wa.me` with a preview like this (first one is preview link, second is the desired behaviour)
![image](https://github.com/devlikeapro/whatsapp-http-api/assets/4376814/628a162f-ad03-44dc-90f7-3db8b4878b1c)
